kmir/src/kmir/kdist/mir-semantics/rt/data.md

```diff
@@ -83,46 +83,6 @@ Constant operands are simply decoded according to their type.
     [preserves-definedness] // valid Map lookup checked
 ```
 
-The code which copies/moves function arguments into the locals of a stack frame works
-in a similar way, but accesses the locals of the _caller_ instead of the locals of the
-current function.
-
-Reading a _Copied_ operand means to simply put it in the K sequence. Obviously, a _Moved_
-local value cannot be read, though, and the value should be initialised.
```

Comment:
Agree with Daniel, we should probably keep some of the deleted text from here to explain the purpose of the new `#traverseProjection` and `operand{Move,Copy}` in general.
The `#readProjection` was actually the first code written for projections, and explains every implemented projection in text before the rule. We should add this back to `#traverseProjection`.

Adding back this documentation can probably be done in a separate PR so we don't lose momentum.